### ISSUE TITLE
NIH clinical center chest X-ray dataset: ChestXray

### ISSUE THREAD

**Issue by 3D-ICE-arXiv** [3D-ICE-arXiv/Papers#59]
ChestX-ray14. The training set contains 86k images while the validation set has 25k X-rays.

![image](https://user-images.githubusercontent.com/73802219/101692593-c5fa3c00-3ab3-11eb-8718-957cd90ed882.png)

■以下を分類
atelectasis 肺拡張不全、無気肺
cardiomegaly 心（臓）肥大
effusion〔体液・血液などの〕浸出
infiltration 浸入、浸透、潜入、浸潤
mass〔物質が一体化した〕塊、まとまり
nodule 《解剖》小（結）節
pneumonia《病理》肺炎
pneumothorax《医》気胸◆肺と胸壁の間にある胸腔内の空気が肺を圧迫して、痛みが起きたり、息が吸いにくくなったりする病気。

![image](https://user-images.githubusercontent.com/73802219/101693206-a283c100-3ab4-11eb-9148-e04b3128d2d5.png)

Wang, X., Peng, Y., Lu, L., Lu, Z., Bagheri, M., Summers, R.M.: ChestX-ray8:
hospital-scale chest x-ray database and benchmarks on weakly-supervised classification
and localization of common thorax diseases. In: Proceedings of the IEEE
Conference on Computer Vision and Pattern Recognition, pp. 2097–2106 (2017)

**Comment by 3D-ICE-arXiv** [comment]
https://github.com/3D-ICE-arXiv/Papers/issues/60

**Comment by 3D-ICE-arXiv** [comment]
https://github.com/3D-ICE-arXiv/Papers/issues/70

**Comment by 3D-ICE-arXiv** [comment]
https://github.com/3D-ICE-arXiv/Papers/issues/85
